documentation/modules/creating-network-mapping.adoc

```diff
@@ -15,18 +15,20 @@ You can create one or more network mappings by using the {project-short} web con
 
 .Procedure
 
-. Click *Mappings*.
-. Click the *Network* tab and then click *Create mapping*.
+. In the {ocp} web console, click *Migration* -> *NetworkMaps for virtualization*.
+. Click *Create NetworkMap*.
 . Complete the following fields:
 
 * *Name*: Enter a name to display in the network mappings list.
 * *Source provider*: Select a source provider.
 * *Target provider*: Select a target provider.
 * *Source networks*: Select a source network.
```

Comment:
should also drop this line, right?

Reply:
I agree this is covered in line 27 below, it should be removed